types/smokehouse.d.ts

```diff
@@ -4,65 +4,71 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
 
- declare module Smokehouse {
-  interface ExpectedLHR {
-    audits: Record<string, any>;
-    requestedUrl: string;
-    finalUrl: string;
-    runWarnings?: Array<string|RegExp>;
-    runtimeError?: {
-      code?: any;
-      message?: any;
-    };
-  }
+import {Artifacts} from './artifacts';
+import Config from './config';
+import LHResult from './lhr';
 
-  export type ExpectedRunnerResult = {
-    lhr: ExpectedLHR,
-    artifacts?: Partial<Record<keyof LH.Artifacts|'_maxChromiumMilestone'|'_minChromiumMilestone', any>>
-    networkRequests?: {length: number};
-  }
+declare global {
```

Comment:
this is equivalent to before (`Smokehouse` is a global type), but as soon the `import` lines were added above the file became a module, so it needed an explicit `declare global`